crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -466,6 +466,24 @@ impl SharedLedgerOps for LedgerDB {
             .get::<ActiveFork>(&())
             .map(|fork| fork.unwrap_or_default())
     }
+
+    /// Get the most recent committed batch
+    /// Returns L2 height.
+    #[instrument(level = "trace", skip(self), err, ret)]
+    fn get_last_commitment_l2_height(&self) -> anyhow::Result<Option<BatchNumber>> {
+        self.db.get::<LastSequencerCommitmentSent>(&())
+    }
+
+    /// Used by the sequencer to record that it has committed to soft confirmations on a given L2 height
+    #[instrument(level = "trace", skip(self), err, ret)]
+    fn set_last_commitment_l2_height(&self, l2_height: BatchNumber) -> Result<(), anyhow::Error> {
+        let mut schema_batch = SchemaBatch::new();
+
+        schema_batch.put::<LastSequencerCommitmentSent>(&(), &l2_height)?;
+        self.db.write_schemas(schema_batch)?;
+
```

Comment:
we can add to the function's documentation that the for  a sequencer  the "last" commitment is set when it is sent but for a full node the last commitment is set when a commitment is read from a finalized DA layer block.